modules/core/02-client/types/msgs_test.go

```diff
@@ -786,10 +791,12 @@ func (suite *TypesTestSuite) TestMsgIBCSoftwareUpgrade_GetSigners() {
 		)
 		suite.Require().NoError(err)
 
+		signers, _, err := suite.chainA.Codec.GetMsgV1Signers(msg)
```

Comment:
ditto on this one being refactored too! we don't need `suite` in this afaik, we can hardcode the chain ID in `clientState` above